packages/client/Usage.md

```diff
@@ -311,7 +311,7 @@ It contains:
 - `meta`: Information about the current page and its cursor.
 
 ```ts
-const page = await xata.db.users.getPaginated();
+const page = await xata.db.users.getPaginated({ pagination: { size: 25, offset: 3 } });
```

Comment:
This example is helpful for understanding what parameters can be passed in here. 

But there is a trades-off against being the code a user would likely want to copy/paste into their project. Especially the `offset: 3` bit.

I don't have any brilliant suggestion of how to fix 🤔. Maybe we could have a pagination options section which shows how a fullish example of the `getPaginated` params?

Reply:
There are more options that can be passed to the `query` methods. Why not adding more examples below?

Reply:
@richardgill https://github.com/xataio/client-ts/pull/542/files#r943475344

Reply:
This looks brilliant :)